pkg/api/entries.go

```diff
@@ -96,6 +96,13 @@ func logEntryFromLeaf(ctx context.Context, signer signature.Signer, tc TrillianC
 	}
 
 	uuid := hex.EncodeToString(leaf.MerkleLeafHash)
+	activeTree := fmt.Sprintf("%x", tc.logID)
```

Comment:
are we guaranteed to only be pulling from the activeTree?

Reply:
Hmmm I assumed the passed `tc` was the one who's log ID would be matching, but I also see a `tid` parameter in this function.

Regardless, if the comment was just on naming, yes, i don't think this is always the active tree. will update